### PR TITLE
Jill(feat)  use tertiary button for date and navigation button in date picker

### PR DESCRIPTION
**Pull request by jillyan**
**What is the problem this PR is trying to solve?**
TCCUI-65 http://guidelines.talend.com/document/92132#/ui-controls/date-time-picker-1561102408
**What is the chosen solution to this problem?**
1. in date view:
 Use tertiary button for date.
 Modify style for Today
2. In header(date view and month-year view):
Replace the navigation button icon by arrow( from chevron)

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss [hunk 1]
```diff
@@ -35,7 +35,9 @@
 
 	.calendar-day {
 		&.today {
-			background-color: $concrete;
+			background-color: tint($lochmara, 90%);
+			color: $lochmara;
+			border: solid 1px $lochmara;
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss]: Remove those css please. The tertiary button is defined in the bootstrap theme, no need to add style

File: packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js [hunk 2]
```diff
@@ -87,11 +88,11 @@ class DateTimeView extends React.PureComponent {
 			leftElement: (
 				<Action
 					aria-label={t('DATEPICKER_MONTH_PREVIOUS', { defaultValue: 'Go to previous month' })}
-					icon="talend-chevron-left"
+					icon="talend-arrow-left"
 					label=""
 					onClick={() => this.goToPreviousMonth()}
 					tabIndex="-1"
-					link
+					className={classNames('btn-tertiary', 'btn-info')}
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js]: The left arrow is not described as a tertiary button in guideline

File: packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js [hunk 3]
```diff
@@ -110,12 +111,12 @@ class DateTimeView extends React.PureComponent {
...
 				<Action
 					aria-label={t('DATEPICKER_MONTH_NEXT', { defaultValue: 'Go to next month' })}
-					icon="talend-chevron-left"
+					icon="talend-arrow-left"
 					iconTransform="rotate-180"
 					label=""
 					onClick={() => this.goToNextMonth()}
 					tabIndex="-1"
-					link
+					className="btn-tertiary btn-info"
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js]: The right arrow is not described as a tertiary button in guideline

File: packages/components/src/DateTimePickers/views/HeaderTitle/HeaderTitle.scss [hunk 4]
```diff
@@ -5,7 +5,7 @@
 	justify-content: center;
 	align-items: center;
 	text-transform: uppercase;
-	font-weight: 600;
+	font-weight: $font-weight-semi-bold;
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/views/HeaderTitle/HeaderTitle.scss]: 👍

File: packages/components/src/DateTimePickers/views/MonthYearView/MonthYearView.component.js [hunk 5]
```diff
@@ -17,7 +17,7 @@ function MonthYearView(props) {
 				icon="talend-arrow-left"
 				onClick={props.onBackClick}
 				tabIndex={props.allowFocus ? 0 : -1}
-				link
+				className="btn-tertiary btn-info"
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/views/MonthYearView/MonthYearView.component.js]: The back button is not described as a tertiary button in guideline

File: packages/components/src/DateTimePickers/views/ViewLayout/ViewLayout.scss [hunk 6]
```diff
@@ -6,16 +6,18 @@
 
 	.header {
 		display: flex;
-		height: 5rem;
+		height: 2.5rem;
```
Comment (jsomsanith-tlnd) [inline packages/components/src/DateTimePickers/views/ViewLayout/ViewLayout.scss]: You should not set a height. 
In the header, define the paddings only (as described in the guideline). The height will be set automatically depending on the paddings and the buttons height.

File: packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js (hunk 2 above)
Comment (jillyan) [reply packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js]: Hi, The left/right arrow is tertiary button in guideline.

File: packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js (hunk 3 above)
Comment (jillyan) [reply packages/components/src/DateTimePickers/views/DateTimeView/DateTimeView.component.js]: Hi, The left/right arrow is tertiary button in guideline.

File: packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss (hunk 1 above)
Comment (jillyan) [reply packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss]: Hi, this is no for tertiary style.
it is for Today.

File: packages/components/src/DateTimePickers/views/MonthYearView/MonthYearView.component.js (hunk 5 above)
Comment (jillyan) [reply packages/components/src/DateTimePickers/views/MonthYearView/MonthYearView.component.js]: From my point of view, it is a tertiary button as the pictures showed in guideline.
Will confirm with ux.

File: packages/components/src/DateTimePickers/views/ViewLayout/ViewLayout.scss (hunk 6 above)
Comment (jillyan) [reply packages/components/src/DateTimePickers/views/ViewLayout/ViewLayout.scss]: updated, Thanks

File: packages/components/src/DateTimePickers/views/MonthYearView/MonthYearView.component.js (hunk 5 above)
Comment (jillyan) [reply packages/components/src/DateTimePickers/views/MonthYearView/MonthYearView.component.js]: Hi Jimmy, checked with ux, it is a tertiary button.
and also the month-year button in date picker.
I have just committed the code to add the style for it.

File: packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss (hunk 1 above)
Comment (jillyan) [reply packages/components/src/DateTimePickers/pickers/DatePicker/DatePicker.scss]: today

**Pull request merged**